03_letters/alanhillgemann/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	runeMap := make(map[rune]int)
```

Comment:
I prefer to avoid `make(…)` except in cases where it's required. In this case you can write:
```go
runeMap := map[rune]int{}
```

Reply:
Fixed